### PR TITLE
Some spelling and grammatical changes.

### PR DESCRIPTION
**Pull request by Alan-Baylis**
Hi Patricio, I hope the following changes are acceptable. Thanks again for your great work!

### CODE REVIEW

File: 07/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Shapes
 
-Finally! We have been building skills for this moment! You have learned most of the GLSL foundations, types and functions. You have practiced your shaping equations over and over. Now is the time to put it all together. You are up for this challenge! In this chapter you'll learn how to draw simple shapes in a parallel procedural way.
+Finally! We have been building skills for this moment! You have learned most of the GLSL foundations, types and functions. You have practiced your shaping equations over and over. Now is the time to put it all together. Are you up for this challenge! In this chapter you'll learn how to draw simple shapes in a parallel procedural way.
```
Comment (wraybowling) [inline 07/README.md]: This wasn't a question; but an encouraging statement.

File: 04/README.md [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
+If you are like me, you will probably want to run shaders directly from the console, in that case you should check out [glslViewer](https://github.com/patriciogonzalezvivo/glslViewer). This application allows you to incorporate shaders into your `bash` scripts or unix pipelines and use it in a similar way to that of [ImageMagick](http://www.imagemagick.org/script/index.php). Also [glslViewer](https://github.com/patriciogonzalezvivo/glslViewer) is a great way to compile shaders on your [Raspberry Pi](https://www.raspberrypi.org/), and the reason why [openFrame.io](http://openframe.io/) use it to display shader artwork. Learn more about this application [here](https://github.com/patriciogonzalezvivo/glslViewer).
```
Comment (wraybowling) [inline 04/README.md]: Make the following improvement
 - the reason why [openFrame.io](http://openframe.io/) **use** it to display
 + the reason why [openFrame.io](http://openframe.io/) **uses** it to display

File: 08/SUMMARY.md [hunk 3]
```diff
@@ -1,2 +1,2 @@
-Matrix is a very powerful tool for manipulating vectors. By mastering how to use matrices, you can freely translate, scale and rotate shapes. Since the technique can be equally applied to anything expressed by vectors, we will look at many more advanced use of matrices later in this book.
```
Comment (wraybowling) [inline 08/SUMMARY.md]: A better fix would be "**A m**atrix is" keeping the subject of a matrix as a tool in the singular. Matrices are referred to later in the paragraph tying the two words together. If you did keep "Matrices are" you should change "tool" to "tools" as well.

File: 09/README.md [hunk 4]
```diff
@@ -101,7 +99,7 @@ Pay close attention to the function ```rotateTilePattern()```, which subdivides
 
 * Comment, uncomment and duplicate lines 69 to 72 to compose new designs.
 
-* Change the black and white triangle for another element like: half circles, rotated squares or lines.
+* Change the black and white triangle for another element: like half circles, rotated squares or lines.
```
Comment (wraybowling) [inline 09/README.md]: Better would be to remove the colon entirely.

File: 10/README.md [hunk 5]
```diff
@@ -22,14 +22,14 @@ Using random can be hard; it is both too chaotic and sometimes not random enough
...
+If you look closely at the random distribution, you will note that the there is some concentration around the middle compared to the edges.
 
 <div class="simpleFunction" data="y = rand(x);
 //y = rand(x)*rand(x);
 //y = sqrt(rand(x));
 //y = pow(rand(x),5.);"></div>
 
-A while ago [Pixelero](https://pixelero.wordpress.com) published an [interesting article about random distribution](https://pixelero.wordpress.com/2008/04/24/various-functions-and-various-distributions-with-mathrandom/). I've added some of the functions he uses in the previous graph for you to play with and see how the distribution can be changed. Uncomment the functions and see what happens.
+A while ago [Pixelero](https://pixelero.wordpress.com) published an [interesting article about random distribution](https://pixelero.wordpress.com/2008/04/24/various-functions-and-various-distributions-with-mathrandom/). I've added some of the functions he uses in the previous graph for you to play with to see how the distribution can be changed. Uncomment the functions and see what happens.
```
Comment (wraybowling) [inline 10/README.md]: Shorten "to play with to see how the distribution can be changed." as "to play with the distribution."

File: 11/SUMMARY.md [hunk 6]
```diff
@@ -1,2 +1,2 @@
-How can we create more natural looking textures like surface of the roads, rocks, trees and clouds? Noise function is the answer.
-Since Ken Perlin invented his first noise algorithm in 80s, the technique has been extensively used throughout computer graphics and simulations. Even if you have never heard of the name, it's not possible you have never seen it. Let's look step by step at how the function is built and works. We also cover more efficient version of the algorithm called simplex noise.
+How can we create more natural looking textures like the surface of roads, rocks, trees and clouds? Noise functions are the answer.
+Since Ken Perlin invented his first noise algorithm in 80's, the technique has been extensively used throughout computer graphics and simulations. Even if you have never heard of the name, it's unlikely you have not seen it. Let's take a step by step look at how the function is built and works. We also cover a more efficient version of the algorithm called simplex noise.
```
Comment (wraybowling) [inline 11/SUMMARY.md]: The actual year that Perlin noise was invented is 1983. Anything less than a year looks like a placeholder.

If the invention(1983)-then-publishing(1985) span makes the use case too... wait for it... noisy, then [see the english stack exchange on the use of apostrophes in the mentioning of decades](https://english.stackexchange.com/a/299512). I'd personally prefer to see "the early-1980s"

File: 07/README.md (hunk 1 above)
Comment (patriciogonzalezvivo) [reply 07/README.md]: It's good to encourage instead of questioning : )

File: 04/README.md (hunk 2 above)
Comment (patriciogonzalezvivo) [reply 04/README.md]: cool

File: 08/SUMMARY.md (hunk 3 above)
Comment (patriciogonzalezvivo) [reply 08/SUMMARY.md]: cool. Seams the change here is directly erasing the paragraph

File: 09/README.md (hunk 4 above)
Comment (patriciogonzalezvivo) [reply 09/README.md]: Mmmmm let me consult Jen on this one

File: 10/README.md (hunk 5 above)
Comment (patriciogonzalezvivo) [reply 10/README.md]: Mmm... not sure. Seams like the 'to' change the meaning

File: 11/SUMMARY.md (hunk 6 above)
Comment (patriciogonzalezvivo) [reply 11/SUMMARY.md]: cool! thanks

**Pull request closed without merging**